### PR TITLE
Improve R console UI experience

### PR DESCRIPTION
**Pull request by brnleehng**
image

- Adding tasks will update every time a new task is added in Batch.
- Details about the job submission

### CODE REVIEW

**Comment by brnleehng** [comment]
Should we also add autoJobDelete?

**Comment by paselem** [comment]
- With the exception of the last section, I think the output looks good.I would only print it if they are NOT using the defaults
- Yes, I would add details about the autoJobDelete
- I would move the 'submitting merge task' output to the next line

**Comment by brnleehng** [comment]
Updated:
image

**Comment by brnleehng** [comment]
The default behavior for autoDeleteJob is to delete the job. 

If the autoDeleteJob is set to FALSE, then we don't get that information about deleting the job/storage containers anyways.

**Comment by paselem** [comment]
Looks good. I would still argue we can remove the Error Count since I can clearly see that in the 'Failed' number.

Remove that last bit and I think we're good.

File: R/doAzureParallel.R [hunk 1]
```diff
@@ -489,9 +489,21 @@ setHttpTraffic <- function(value = FALSE) {
     }
   }
 
-  cat("Job Summary: ", fill = TRUE)
+  cat(strrep('=', options("width")), fill = TRUE)
```
Comment (paselem) [inline R/doAzureParallel.R]: I would factor this entire printing section into it's own method

**Comment by brnleehng** [comment]
Removed error count

**Pull request merged**